test/EVM/Test/Tracing.hs

```diff
@@ -145,22 +145,24 @@ data EVMToolEnv = EVMToolEnv
   { coinbase    :: Addr
   , timestamp   :: Expr EWord
   , number      :: W256
-  , prevRandao  :: W256
   , gasLimit    :: Data.Word.Word64
   , baseFee     :: W256
   , maxCodeSize :: W256
   , schedule    :: FeeSchedule Data.Word.Word64
   , blockHashes :: Map.Map Int W256
+  , withdrawals :: [(Addr)]
+  , currentRandom :: W256
   } deriving (Show, Generic)
 
 instance JSON.ToJSON EVMToolEnv where
   toJSON b = JSON.object [ ("currentCoinBase"  , (JSON.toJSON $ b.coinbase))
-                         , ("currentDifficulty", (JSON.toJSON $ b.prevRandao))
                          , ("currentGasLimit"  , (JSON.toJSON ("0x" ++ showHex (into @Integer b.gasLimit) "")))
                          , ("currentNumber"    , (JSON.toJSON $ b.number))
                          , ("currentTimestamp" , (JSON.toJSON tstamp))
                          , ("currentBaseFee"   , (JSON.toJSON $ b.baseFee))
                          , ("blockHashes"      , (JSON.toJSON $ b.blockHashes))
+                         , ("withdrawals"      , (JSON.toJSON $ b.withdrawals))
```

Comment:
not strictly related to this PR but we don't need `$` here

Reply:
Fixed all the `$`-s, thanks!